game_of_thrones/app/controllers/characters_controller.rb

```diff
@@ -0,0 +1,49 @@
+class CharactersController < ApplicationController
+  def index
+    @characters = Character.all
+  end
+
+  def show
+    @character = Character.find(params[:id])
+  end
+
+  def new
+    @character = Character.new
+    @houses = House.all.map do |house|
+      [house.name, house.id]
+    end
+  end
+
+  def create
+    character = Character.new(character_params)
+    character.house_id = params[:house_id]
+    character.save!
+    redirect_to character
+  end
+
+  def edit
```

Comment:
check out my comments in your `edit.html.erb`

you might want to call your  `@houses` `@houses_for_select`